docs/advanced_usage/8_logging.md

```diff
@@ -27,6 +27,113 @@ The table shows you the specific levels:
+```python
+Scenario(
+    configspace = some_configspace,
+    name = 'experiment_name',
+    output_directory = Path('some_directory'),
+    ...
+)
+```
+Notably, if an output already exists at `./some_directory/experiment_name/seed`, the behavior is determined by the overwrite parameter in the [facade's][smac/facade/abstract_facade] settings. This parameter specifies whether to continue the previous run (default) or start a new run.
+
+The output is split into four different log files, and a copy of the utilized [Configuration Space of the ConfigSpace library](https://automl.github.io/ConfigSpace/latest/).
+
+### intensifier.json
+The [intensification][Intensification] is logged in `intensifier.json` and has the following structure:
+
+```json
+{
+  "incumbent_ids": [
+    65
+  ],
+  "rejected_config_ids": [
+    1,
+  ],
+  "incumbents_changed": 2,
+  "trajectory": [
+    {
+      "config_ids": [
+        1
+      ],
+      "costs": [
+        0.45706284046173096
+      ],
+      "trial": 1,
+      "walltime": 0.029736042022705078
+    },
+    #...
+  ],
+  "state": {
+    "tracker": {},
+    "next_bracket": 0
+  }
+}
+```
+
+### optimization.json
+The optimization process is portrayed in `optimization.json` with the following structure
+
+```json
+{
+  "used_walltime": 184.87366724014282,
+  "used_target_function_walltime": 20.229533672332764,
+  "last_update": 1732703596.5609574,
+  "finished": false
+}
+``` 
+### runhistory.json
+The runhistory.json in split into four parts. `stats`, `data`, `configs`, and `config_origins`.
+`stats` contains overall broad stats on the different evaluated configurations:
+```json
+  "stats": {
+    "submitted": 73,
+    "finished": 73,
+    "running": 0
+  },
+```
+
+`data` contains a list of entries, one for each configuration.
+```json
+  "data": [
+    [
+      1,                            # config_id
+      null,                         # instance or None
+      209652396,                    # seed or None
+      null,                         # budget or None
+      5.4345623938566385,           # cost
+      6.699562072753906e-05,        # time
+      6.299999999992423e-05,        # cpu_time
+      1,                            # status
+      1733133181.2144582,           # start_time
+      1733133181.21695,             # end_time
+      {}                            # additional_info
+    ],
+    ...
+  ]
+```
+
+`configs` contains a human readable dictionary of configurations. Note that this list starts by one, therefore the 0 entry from above is mapped to 1.
+```json
+  "configs": {
+    "1": {
+      "x": -2.3312147893012
+    },
+```
+
+Lastly, `config_origins` contains the origin of a configuration. 
```

Comment:
add, that origins can be the from the initial design or from different acquisition function maximization techniques